src/client/components/ProductsContainer.js

```diff
@@ -10,16 +10,21 @@ const ProductsContainer = ({ isArchive, products }) => (
         {isArchive ? 'Orders history' : 'Products list'}
       </h2>
       <span className="products__info">
-        {isArchive ? 'Remove item' : 'Mark as ordered'}
+        {isArchive ? 'Restore item' : 'Mark as ordered'}
       </span>
     </header>
+    {!products.length && (
```

Comment:
move that functionality to ProductList - if it will get empty list than it will display emptyText